src/pages/search/advanced_search/AdvancedSearchPage.tsx

```diff
@@ -61,100 +74,143 @@ export const AdvancedSearchPage = () => {
-      <Box
+    <Grid container sx={{ bgcolor: 'secondary.main' }}>
+      <Grid
+        item
+        container
+        rowGap={1}
         sx={{
-          display: 'flex',
-          flexDirection: 'column',
-          gap: '0.75rem',
           mx: { xs: '0.5rem', md: 0 },
           mb: '0.75rem',
+          p: '1rem',
         }}>
-        <Box sx={{ display: 'flex', gap: '0.5rem', flexWrap: 'wrap' }}>
-          <SearchForm
-            sx={{ flex: '1 0 15rem' }}
-            paramName={ResultParam.Title}
-            placeholder={t('search.search_for_title')}
-          />
-          <ExportResultsButton searchParams={params} />
-        </Box>
-
-        <Box sx={{ display: 'flex', alignItems: 'center', gap: '1rem', flexWrap: 'wrap' }}>
-          <PublicationDateIntervalFilter />
-
-          {showFilterDivider && <Divider orientation="vertical" flexItem />}
-
-          <OrganizationFilters topLevelOrganizationId={topLevelOrganizationId} unitId={unitId} />
-
-          {showFilterDivider && <Divider orientation="vertical" flexItem />}
-
-          <SearchForm paramName={ResultParam.ContributorName} placeholder={t('search.search_for_contributor')} />
-
-          {showFilterDivider && <Divider orientation="vertical" flexItem />}
-
-          <FundingSourceFilter />
-
-          {showFilterDivider && <Divider orientation="vertical" flexItem />}
-
-          <SearchForm
-            paramName={ResultParam.FundingIdentifier}
-            placeholder={t('search.search_for_funding_identifier')}
-          />
-
-          {showFilterDivider && <Divider orientation="vertical" flexItem />}
-
-          <PublisherFilter />
-
-          {showFilterDivider && <Divider orientation="vertical" flexItem />}
-
-          <JournalFilter />
-
-          {showFilterDivider && <Divider orientation="vertical" flexItem />}
-
-          <SeriesFilter />
-
-          {showFilterDivider && <Divider orientation="vertical" flexItem />}
-
-          <LanguageFilter />
-
-          {showFilterDivider && <Divider orientation="vertical" flexItem />}
-
-          <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: '0.25rem' }}>
-            {categoryShould.slice(0, 3).map((category) => (
-              <CategoryChip
-                key={category}
-                category={{
-                  value: category,
-                  text: t(`registration.publication_types.${category}`),
-                  selected: true,
-                }}
-                onClickChip={toggleCategoryFilter}
-              />
-            ))}
-            {categoryShould.length > 3 ? (
-              <Chip
-                label={t('common.x_others', { count: categoryShould.length - 3 })}
-                variant="filled"
-                color="primary"
-                onClick={toggleCategoryFilter}
+        <Typography variant="h3">{t('search.advanced_search.advanced_search')}</Typography>
+        <Grid item xs={12} sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
```

Comment:
har som sagt ikke brukt Grid selv, men har forstått det som at det er en slags opionated `flex`, så virker rart å definere en flex i stylingen der?
```suggestion
        <Grid item xs={12} direction="column" gap={1}>
```